### ISSUE TITLE
Email Not Sending Out - Unable to Setup docker-compose

### ISSUE THREAD

**Issue by rpaGuyai** [bionic-gpt/bionic-gpt#474]
### Describe your issue

I have followed the instructions - https://bionic-gpt.com/docs/administration/configuring-email/ to configure email.
Updated docker-compose.yml file with the details and ran command 
1. docker-compose build d
2. docker-compose up

Output Below

![image](https://github.com/bionic-gpt/bionic-gpt/assets/154881376/38015be7-212c-467d-8c6b-9e98ad1163b0)

Output in Docker

![image](https://github.com/bionic-gpt/bionic-gpt/assets/154881376/c57297f3-bb4b-4b22-9a4b-45475d9f1938)

Still Unable to receive email invites.

### Describe your setup/startup scripts

Setup environment variables in docker-compose,yml

Execute command - dcoker-compose.- up
.Followed the instruction to setup email.

Run - Docker-compose -up

### Steps to reproduce

.Followed the instruction to setup email.

Run - Docker-compose -up

Executed command 
docker -compose - build
docker-compose -up

Still couldn't send out emails

**I am using Docker For windows which does not have any docker-comose.yml file by default.**

**Update - 16/05/2-024**

![image](https://github.com/bionic-gpt/bionic-gpt/assets/154881376/d81bb684-8ee8-41f7-974d-c800f4d11de4)

Not sure why it says from email as 'support@application.com', either ways, still couldn't receive any email.

![image](https://github.com/bionic-gpt/bionic-gpt/assets/154881376/43f2376b-0212-4a6c-a1e6-1473c4e1f97a)

My docker-compose.yml file

![image](https://github.com/bionic-gpt/bionic-gpt/assets/154881376/316ce727-aed9-4866-9872-b9d1a4e2483d)

**Comment by rpaGuyai** [comment]
@9876691 Please could you assist on this, it will be very helpful, we have tried quite a few solutions but still couldn't make it work.

**Comment by 9876691** [comment]
Hi, the documentation for email hasn't been updated since we moved to a Kubernetes based install.

I'll work on it over the weekend and let you know.

**Comment by 9876691** [comment]
Documentation has been updated to use a K8s secret.

**Comment by rpaGuyai** [comment]
Hi @9876691 I tried the steps in the document to configure email - updated as of 21/05/2024.
Still same issue, we are unable to receive any email, also, same as earlier, the log says from email as "support@application.com".

1. We updated the SMTP using below
![image](https://github.com/bionic-gpt/bionic-gpt/assets/154881376/190e2d36-3c40-4f39-b6e7-85bd41835009)

2. Restarted the Bionic Server using below
![image](https://github.com/bionic-gpt/bionic-gpt/assets/154881376/a46edbe4-e0e0-4462-8899-175f5556bbcb)

3. Tried sending email - same issue, cannot receive email, logs still referring to support@application.com
4. Manually re-started the docker, then receiving below error, to resolve this error, again we need to reset Kubernetes and re-install things.

![image](https://github.com/bionic-gpt/bionic-gpt/assets/154881376/a08bd9e3-7029-4bce-a8aa-31e68f7e60dd)
